components/plurals/src/lib.rs

```diff
@@ -276,10 +276,7 @@ impl PluralCategory {
 /// [`ICU4X`]: ../icu/index.html
 /// [`Plural Type`]: PluralRuleType
 /// [`Plural Category`]: PluralCategory
-pub struct PluralRules {
-    _locale: Locale,
-    rules: DataPayload<ErasedPluralRulesV1Marker>,
-}
+pub struct PluralRules(DataPayload<ErasedPluralRulesV1Marker>);
```

Comment:
`nit`: Can you deref here and then use `self.get()` instead of `self.0.get()` everywhere?